src/main/resources/hw16-screenshots/hw17.sql

```diff
@@ -0,0 +1,149 @@
+inner join customers c on o.cust = c.cust_num
+inner join salesreps s on c.cust_rep = s.empl_num and o.amount > 700;
+
+--List orders above $ 700
+--including the name of the customer who placed the order
+--and the name of the employee assigned to that customer, as well as the office where the employee works
+select order_num, amount, c.company, s.name, ofc.city from orders o
+inner join customers c on o.cust = c.cust_num
+inner join salesreps s on c.cust_rep = s.empl_num 
+inner join offices ofc on s.rep_office = ofc.office and o.amount > 700;
+
+--step 28
+
+--Find all orders received on the day when a new employee was hired
+select order_num, amount, order_date, s.name, s.hire_date from orders o
+inner join salesreps s on o.order_date = s.hire_date;
+
+--Get all the combinations of employees offices
+--where the planned sales volume of the employee is greater than the plan of any office.
+select name, quota, o.city, o.target from salesreps s
+inner join offices o on s.quota > o.target;
+
+--Show each employee's name, office, and sales
+select name, s.sales, o.city from salesreps s
+inner join offices o on s.rep_office = o.office;
+
+--Show the names of cities in which actual sales exceed planned
+select city, sales from offices where sales > target;
+
+--Show the names of employees whose sales are above $ 1000.
+select name, sales from salesreps where sales > 200;
+
+--Report all information about employees and offices where they work.
+select * from salesreps
+inner join offices on rep_office = office;
+select s.*, o.city, o.region from salesreps s
+inner join offices o on rep_office = office;
+
+--step 30
+
+--list the names of all employees and their managers
+select s1.name, s2.name from salesreps s1
+inner join salesreps s2 on s1.manager = s2.empl_num;
+
+--Display a list of employees whose plans exceed the plans of their leaders
+select s.name, s.quota, mgrs.quota from salesreps s
+inner join salesreps mgrs on s.manager = mgrs.empl_num and s.quota > mgrs.quota;
+
+--Display a list of employees who work with their managers in various offices
+--including the names and offices of both employees and managers.
+select s.name, so.city, mgrs.name, mgrso.city from salesreps s
+inner join offices so on s.rep_office = so.office
+inner join salesreps mgrs on s.manager = mgrs.empl_num and s.rep_office <> mgrs.rep_office
+inner join offices mgrso on mgrs.rep_office = mgrso.office;
+
+--step 31
+
+--Output averege quota and avg sales from salesreps
+select avg(quota), avg(sales) from salesreps;
+
+--Output averege plan-execution from salesreps
+select avg(100 * (sales/quota)) from salesreps;
+select avg(100 * (sales/quota)) procent from salesreps;
+
+--Output sum of quota and sales from salesreps
+select sum(quota), sum (sales) from salesreps;
+
+--Output amount of all offers recived by Dima Malkov
+select sum(amount) from salesreps s
+inner join orders o on s.name = '���� �������' and o.rep = s.empl_num;
```

Comment:
Kirillica fucked up) Try to upload file thru the IDEA

Reply:
i uploaded it via idea